### ISSUE TITLE
Warning: Got ack for non-current block (2 != 4)

### ISSUE THREAD

**Issue by dbrgn** [dbrgn/iboardbot-web#6]
Apparently the bot can get stuck with the message above.

**Comment by dbrgn** [comment]
This is the second print job in a row:

```
< CD Polling serial...
< CL STATUS=ACK&NUM=3
< CD Bytes read: 0
< !ERROR: M Timeout
< 
< CD Polling serial...
< CL STATUS=ACK&NUM=3
Starting scheduled print
< CD Bytes read: 0
< !ERROR: M Timeout
< 
< CD Polling serial...
< CL STATUS=ACK&NUM=3
> Print a block
< CD Bytes read: 768
< CD Processing packet... Lines: 256
< CD MORE BLOCKS!
< CD Start block: 1
< !START DRAW
< SE
< !FINISH! time:129449
< 
< CD Polling serial...
< CL STATUS=ACK&NUM=1
> Print a block
< CD Bytes read: 768
< CD Processing packet... Lines: 256
< CD MORE BLOCKS!
< CD Start block: 2
< !FINISH! time:37055
< 
< CD Polling serial...
< CL STATUS=ACK&NUM=2
Warning: Got ack for non-current block (2 != 5)
< CD Bytes read: 0
< !ERROR: M Timeout
< 
< CD Polling serial...
< CL STATUS=ACK&NUM=2
Warning: Got ack for non-current block (2 != 5)
< CD Bytes read: 0
< !ERROR: M Timeout
< 
< CD Polling serial...
< CL STATUS=ACK&NUM=2
Warning: Got ack for non-current block (2 != 5)
< CD Bytes read: 0
< !ERROR: M Timeout
< 
```

**Comment by dbrgn** [comment]
Should be fixed in 5a7e442.
